docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -52,18 +51,18 @@ stream should be rendered. We'll discuss this in depth in the next section thoug
 ### Customising response rendering mode
 
 Since it is not always possible to directly and confidently answer the question of how a stream of `T` should look on 
-the wire, the `EntityStreamingSupport` traits come into play and allow fine-tuning the streams rendered representation.
+the wire, the `EntityStreamingSupport` traits come into play and allow fine-tuning the stream's rendered representation.
 
 For example, in case of JSON Streaming, there isn't really one standard about rendering the response. Some APIs prefer
-to render multiple JSON objects in a line-by-line fashion (Twitter's streaming APIs for example), while others simply return
+to render multiple JSON objects in a line by line fashion (Twitter's streaming APIs for example), while others simply return
```

Comment:
[This](https://www.dict.cc/englisch-deutsch/line+by+line.html) tells me, that both versions are possible. Let's go with the hyphenated variant.